DigitalLearningSolutions.Web/Views/Frameworks/Developer/_SignpostingPaginationNav.cshtml

```diff
@@ -0,0 +1,45 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Frameworks
+@model CompetencyResourceSignpostingViewModel
+
+<nav class="nhsuk-pagination" role="navigation" aria-label="Pagination">
+  <form method="post" role="search" asp-controller="Frameworks" asp-action="GoToPage">
+    <div class="nhsuk-pagination__list">
+      <div class="pagination-button-container">
+        <div class="nhsuk-pagination-item--previous" @(Model.Page == 1 ? "hidden" : "")>
+          <button class="nhsuk-pagination__link nhsuk-pagination__link--prev" type="submit"
+                  asp-action="GoToPreviousPage"
+                  asp-controller="Frameworks">
+            <svg class="nhsuk-icon nhsuk-icon__arrow-left" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+              <path d="M4.1 12.3l2.7 3c.2.2.5.2.7 0 .1-.1.1-.2.1-.3v-2h11c.6 0 1-.4 1-1s-.4-1-1-1h-11V9c0-.2-.1-.4-.3-.5h-.2c-.1 0-.3.1-.4.2l-2.7 3c0 .2 0 .4.1.6z"></path>
+            </svg>
+            <span class="nhsuk-pagination__title">Previous</span>
+          </button>
+        </div>
+      </div>
+      <div class="page-indicator-container">
+        <span aria-live="polite" aria-atomic="true">
+          <span class="nhsuk-u-visually-hidden">page </span>
+          <span id="page-indicator">@Model.Page of @Model.TotalPages</span>
+        </span>
+      </div>
+      <div class="pagination-button-container">
+        <div class="nhsuk-pagination-item--next" @(Model.Page < Model.TotalPages ? "" : "hidden")>
+          <button class="nhsuk-pagination__link nhsuk-pagination__link--next" type="submit"
+                  asp-action="GoToNextPage"
+                  asp-controller="Frameworks">
+            <span class="nhsuk-pagination__title">Next</span>
+            <svg class="nhsuk-icon nhsuk-icon__arrow-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+              <path d="M19.6 11.66l-2.73-3A.51.51 0 0 0 16 9v2H5a1 1 0 0 0 0 2h11v2a.5.5 0 0 0 .32.46.39.39 0 0 0 .18 0 .52.52 0 0 0 .37-.16l2.73-3a.5.5 0 0 0 0-.64z"></path>
+            </svg>
+          </button>
+        </div>
+      </div>
+    </div>
+    <input id="FrameworkId" name="FrameworkId" type="hidden" value="@Model.FrameworkId" />
```

Comment:
Not sure these are necessary if we move to using a GET method to retrieve data rather than posting it from this page. See previous comment. But if we do keep them, it might be tidier to use the `@Html.HiddenFor(m => m.FrameworkId)` helper used in other views?

Reply:
Originally they were HiddenFor but this is different than input hidden. HiddenFor ignores user changes on post back as described on [this post](https://stackoverflow.com/questions/20657706/mvc-4-html-hiddenfor-are-not-updating-on-a-postback/23200608) and this was a problem when search text is edited by user.

Reply:
Makes sense. We may not need them if we change the summary method to GET rather than post, these will be route values in a link button rather than form inputs if so.